kolibri/plugins/management/assets/src/device_management/views/manage-permissions-page/search-bar.vue

```diff
@@ -0,0 +1,109 @@
+<template>
```

Comment:
Might make sense to share this component between the Facility and Device SPAs?

Reply:
I dunno. I think this is component is going to go away anyhow after `k-filter-textbox` is ready.

Reply:
Righto - if it's only temporary, no worries.

Reply:
it's ready! https://github.com/learningequality/kolibri/pull/1944 :)

let's update this PR to use it?